blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/CalendarClock.java

```diff
@@ -0,0 +1,21 @@
+package com.rackspacecloud.blueflood.utils;
+
+import org.joda.time.Instant;
+
+import java.util.Calendar;
+
+public class CalendarClock implements Clock {
```

Comment:
We currently use the ``DefaultClock`` in ``ScheduleContext`` and ``ShardStateManager``. I dont see any reason why someone would mess with joda-time to configure a different ``millisProvider``. But if they do, they better evaluate the impact properly.

I think its pretty safe to use DefaultClockImpl.